Cotton Classing Database File Import Plugin/DataMappers/NDBRecordMapper.cs

```diff
@@ -0,0 +1,191 @@
+﻿using AgGateway.ADAPT.ApplicationDataModel.ADM;
+using AgGateway.ADAPT.ApplicationDataModel.Common;
+using AgGateway.ADAPT.ApplicationDataModel.Logistics;
+using AgGateway.ADAPT.ApplicationDataModel.Products;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using AgGateway.ADAPT.ApplicationDataModel.Documents;
+
+namespace CottonClassingPlugin.DataMappers
+{
+    public class NDBRecordMapper
+    {
+        public static AgGateway.ADAPT.ApplicationDataModel.Documents.ObsCollection MapNDBRecordObsCollection(PublisherDataModel.NDBRecord ndbRecord, List<AgGateway.ADAPT.ApplicationDataModel.Documents.Obs> _obs, int obsCollectionID, AgGateway.ADAPT.ApplicationDataModel.Documents.ObsCollection obsCollection, ObsDataset obsDataset, ApplicationDataModel adm)
+        {
+            int obsIndex = 1;
+            obsCollection.OMSourceId = obsCollectionID;
+
+            Obs OfficialColorGrade = new Obs();
```

Comment:
Another stylistic comment: Pascal casing is generally not used for variables; instead prefer camel casing.